backend/controller/cronjobs/cronjobs_test.go

```diff
@@ -1,5 +1,3 @@
-//go:build !integration
```

Comment:
@matt2e I have `integration` added to my build tags so that I can edit integration tests with completion (which I recommend), and this negation breaks completion in these files. For the purposes of running the integration tests we can either just run these, or explicitly select just the tests in files with the integration tag.